ci/run-k8s-e2e-tests.sh

```diff
@@ -31,8 +31,8 @@ RUN_NETWORK_POLICY=false
 RUN_E2E_FOCUS=""
 RUN_E2E_SKIP=""
 KUBECONFIG_OPTION=""
-DEFAULT_E2E_CONFORMANCE_FOCUS="\[Conformance\]"
-DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
+DEFAULT_E2E_CONFORMANCE_FOCUS="\[sig-network\]|\[Conformance\]"
```

Comment:
Conformance test usually selects about 200+ cases, and NetworkPolicy test usually selects 80+ cases. In my experience the NetworkPolicy tests usually use less time than Conformance test. Is it better to run sig-network tests together with NetworkPolicy tests?

Reply:
On upstream CI, conformance test took 30min and networkpolicy test took about 1 hour. Since the sig-network test cost another 1 hour, I guess it's better to add it to conformance test?  @edwardbadboy

Reply:
After discussion, we agreed to add sig-network to conformance job.